programs/fileio.c

```diff
@@ -487,6 +489,10 @@ void FIO_setLdmHashRateLog(FIO_prefs_t* const prefs, int ldmHashRateLog) {
     prefs->ldmHashRateLog = ldmHashRateLog;
 }
 
+void FIO_setDiffFromMode(FIO_prefs_t* const prefs, int diffFromMode)
+{
+    prefs->diffFromMode = diffFromMode;
```

Comment:
minor : 
`diffFromMode` is essentially a flag,
so it's better to consider it a `0` / `1` value, and not have to worry afterwards that the "real" value it carries might be different.

One way to ensure that outcome is : 
`prefs->diffFromMode = (diffFromMode != 0);`